script.js

```diff
@@ -37,21 +35,24 @@
 // Write your solution below
 // ====================================================================
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+// var userInput = prompt("Enter a number");
+
+// function ordinalSuffix(userInput) {
+//     var modulus1 = userInput % 10,
+//         modulus2 = userInput % 100;
+//     if (modulus1 == 1 && modulus2 != 11) {
+//         return userInput + "st";
+//     }
+//     if (modulus1 == 2 && modulus2 != 12) {
+//         return userInput + "nd";
+//     }
+//     if (modulus1 == 3 && modulus2 != 11) {
+//         return userInput + "rd";
+//     }
+//     return userInput + "th";
+// }
+
+// console.log(ordinalSuffix(userInput));
```

Comment:
Just a comment since I noticed you used multiple if statements here, and elsewhere you've used if-else statements...  what made you choose multiple if statements over if-else statements here?